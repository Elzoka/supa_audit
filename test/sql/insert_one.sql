create table public.members(
    id int primary key,
    name text not null
);

create trigger t
    before insert or update or delete
    on public.members
    for each row
    execute procedure audit.change_trigger();

insert into public.members(id, name)
values (1, 'foo');

select
    id,
    record_id,
    old_record_id,
    op,
    entity_oid::regclass,
    record
from
    audit.record_version_history;
