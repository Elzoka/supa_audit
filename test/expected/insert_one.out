create table public.members(
    id int primary key,
    name text not null
);
create trigger t
    before insert or update or delete
    on public.members
    for each row
    execute procedure audit.change_trigger();
insert into public.members(id, name)
values (1, 'foo');
select 
    id,
    record_id,
    old_record_id,
    op,
    entity_oid::regclass,
    record
from 
    audit.record_version_history;
 id |              record_id               | old_record_id |   op   | entity_oid |          record          
----+--------------------------------------+---------------+--------+------------+--------------------------
  1 | dd76759f-e596-510c-80a6-209bb6ee0d7d |               | INSERT | members    | {"id": 1, "name": "foo"}
(1 row)

